tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.set_props.ksh

```diff
@@ -0,0 +1,40 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 by Delphix. All rights reserved.
+#
+
+. $STF_SUITE/tests/functional/channel_program/channel_common.kshlib
+
+#
+# DESCRIPTION:
+#       Getting number props should work correctly on filesystems,
+#	snapshots and volumes.
+#
+
+verify_runnable "global"
+
+fs=$TESTPOOL/$TESTFS/testchild
+
+function cleanup
+{
+	datasetexists $fs && log_must zfs destroy -R $fs
```

Comment:
Can you update this to the following, the wrapper adds a little additional test resiliency. 

```sh
        destroy_dataset $fs "-R"
```

Reply:
Done